documentation/Functions.md

```diff
@@ -349,20 +349,46 @@ path: The URL path on which is checked if the server is running
 If the tutorial should be tested on the console environment, you have to specify a port.
 ***
 
-### nextKatacodaStep <a name="nextKatacodaStep"></a>
+### displayContent <a name="displayContent"></a>
 #### parameter
-1. The title of the step.
-2. An array of json objects with files, content, or images to be rendered within the katacoda step.
+1. The title of the step. 
+2. An array of json objects with files, content, or images to be rendered within the katacoda step. The use for this function is to display an image and some descriptive text. No Katacoda Syntax is allowed in the files or the content!
 3. (Optional) Path to the current directory where the user is located (relative to the workspace directory). Only needed if the directory is changed within this step.
 #### example 
-nextKatacodaStep("Step title", [{ "file": "files/description.md" }, { "content": "This is just plain content." }, { "image": "files/image.png" }])
+display("Step title", [{ "file": "files/description.asciidoc" }, { "content": "This is just plain content." }, { "image": "files/image.png" }])
 
 #### Details
 Available attributes in the json objects:
 
-file: Path to a file whose content is to be displayed in the katacoda step (e.g. .md or .txt file).
-content: Plain text to be displayed in the katacoda step.
-image: Path to an image to be displayed in the katacoda step.
+1. file: Path to a file whose content is to be displayed in the katacoda step (e.g. .asciidoc or .txt file). The file should be following the formating of asciidoc files. 
```

Comment:
Katacoda with capital first letter. Check also in the rest of the text
formatting

Reply:
Changed it